README.md

```diff
@@ -13,3 +13,14 @@ Check our [examples repo](https://github.com/aws-samples/aws-bootstrap-kit-examp
 As of today we expose only one global construct :
 
 * [AWS BootstrapKit](./source/aws-bootstrap-kit/README.md)
+
+## Clean up accounts
+
+AWS Bootstrap Kit creates accounts (e.g CICD, DNS, Staging) through AWS Organization. Behind the scene, we use CloudFormation Custom Resource and AWS Lambda to create to call [CreateAccount API](https://docs.aws.amazon.com/organizations/latest/APIReference/API_CreateAccount.html). 
+
+However, there is no API to delete and remove accounts. Thus, the accounts cannot be cleaned up automatically in via Custom Resources. We need to delete these accounts manually. The steps below explain how to clean up those accounts. You have to repeat these step for each account one by one.
+
+1. **Ensure that you have access email address of the member account** Without access to email address, you cannot close the account. If that is the case, create [a Service Control Policy (SCP)](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html) with deny * on all resources and apply it to all accounts to disable all access.
```

Comment:
Ensure that you have access to the email address ...

Reply:
Fixed.